requirements.txt

```diff
@@ -20,4 +20,6 @@ SQLAlchemy-Utils==0.32.1
 xmlrunner==1.7.7
 pytest==2.9.2
 pytest-cov==2.3.1
-suds-jurko==0.6
+# suds is outdated but more compatible with FSRS than other libs. Pin a
+# version here
+-e git+https://github.com/andersinno/suds-jurko.git@44e27560b328ec6482459ea2d75516849dc209ff#egg=suds
```

Comment:
Thanks for the explanation in the commit message. This change would've definitely been a head-scratcher otherwise.